### PR TITLE
Added column for 'temporary' issues in the recent labels table for Admin interface

### PR DESCRIPTION
**Pull request by manaswis**

### CODE REVIEW

**Comment by jonfroehlich** [comment]
To be clear, this 'temporary column' is not actually a _temporary_ column, it's a permanent column describing the temporary status of a label (as labels can be marked as temporaroy...e.g., in the case of construction).

**Comment by manaswis** [comment]
Yes, that's correct. Updated the pull request title.

File: app/models/label/LabelTable.scala [hunk 1]
```diff
@@ -58,7 +58,8 @@ object LabelTable {
 
   case class LabelMetadata(labelId: Int, auditTaskId: Int, userId: String, username: String,
                            timestamp: java.sql.Timestamp,
-                           labelTypeDesc: String, severity: Option[Int], description: Option[String])
+                           labelTypeDesc: String, severity: Option[Int],
+                           temporary: Boolean, description: Option[String])
```
Comment (kotarohara) [inline app/models/label/LabelTable.scala]: Can you wrap temporary with `Option`?

File: app/models/label/LabelTable.scala [hunk 2]
```diff
@@ -163,19 +169,24 @@ object LabelTable {
    * Date: Sep 2, 2016
    */
   def selectTopLabelsAndMetadataByUser(n: Int, userId: UUID): List[LabelMetadata] = db.withSession { implicit session =>
-    val selectQuery = Q.queryNA[(Int, Int, String, String, java.sql.Timestamp, String, Option[Int], Option[String])](
+    val selectQuery = Q.queryNA[(Int, Int, String, String, java.sql.Timestamp, String, Option[Int],
+                                  Boolean, Option[String])](
```
Comment (kotarohara) [inline app/models/label/LabelTable.scala]: Once you update the case class you should change Boolean to Option[Boolean]

Comment (manaswis) [inline app/models/label/LabelTable.scala]: I actually handled that in the query. The query will _always_ return a true or a false.

File: app/models/label/LabelTable.scala [hunk 3]
```diff
@@ -135,19 +136,24 @@ object LabelTable {
...
+    val selectQuery = Q.queryNA[(Int, Int, String, String, java.sql.Timestamp, String, Option[Int], Boolean,
+                                  Option[String])](
       """SELECT lb1.label_id, lb1.audit_task_id, u.user_id, u.username, ati.timestamp,
-        |       lb_big.label_type_desc, lb_big.severity, lb_big.description
+        |       lb_big.label_type_desc, lb_big.severity, lb_big.temp_problem, lb_big.description
         |	FROM sidewalk.label as lb1, sidewalk.audit_task as at, sidewalk.audit_task_interaction as ati,
         |       sidewalk.user as u,
-        |				(SELECT lb.label_id, lbt.description as label_type_desc, sev.severity, prob_desc.description
+        |				(SELECT lb.label_id, lbt.description as label_type_desc, sev.severity,
+        |               COALESCE(prob_temp.temporary_problem,'FALSE') as temp_problem,
```
Comment (manaswis) [inline app/models/label/LabelTable.scala]: True/False return happens here.

File: app/models/label/LabelTable.scala (hunk 2 above)
Comment (kotarohara) [inline app/models/label/LabelTable.scala]: I see. Great!

**Comment by jonfroehlich** [comment]
You guys rock! Go team!

@kotarohara, thank you for so carefully reviewing the pull requests before merge. This is important to help maintain the quality of our system but also to train @Soben713 and @manaswis. Yay! :)

**Pull request merged**